examples/python/s2ag-datasets.py

```diff
@@ -0,0 +1,60 @@
+"""
+Sample usage of Semantic Scholar Academic Graph Datasets API
+https://api.semanticscholar.org/api-docs/datasets
+"""
+import requests
+import urllib
+import json
+import os
+
+# Get info about the latest release
+latest_release = requests.get("http://api.semanticscholar.org/datasets/v1/release/latest").json()
+print(latest_release['README'])
+print(latest_release['release_id'])
+
+# Get info about past releases
+dataset_ids = requests.get("http://api.semanticscholar.org/datasets/v1/release").json()
+earliest_release = requests.get(f"http://api.semanticscholar.org/datasets/v1/release/{dataset_ids[0]}").json()
+
+# Print names of datasets in the release
```

Comment:
Nit: consider printing: `"\n".join(s"name: {d['name']}, description: {d['README']}")` so that new users can make sense of what the different datasets provide.